chart/values.yaml

```
@@ -4,17 +4,18 @@
 
 manager:
```

continuing the nit with proposed improvement:
```suggestion
manager:
  annotations:
    sidecar.istio.io/inject: "false"
```